packages/Webkul/Shop/src/Resources/lang/de/app.php

```diff
@@ -458,6 +458,8 @@
                 'success' => 'Artikel wurde erfolgreich zum Warenkorb hinzugefügt',
                 'success-remove' => 'Artikel wurde erfolgreich aus dem Warenkorb entfernt',
                 'error-add' => 'Artikel kann nicht zum Warenkorb hinzugefügt werden. Bitte versuchen Sie es später erneut',
+                'inactive' => 'Ein Artikel ist inaktiv und wurde aus dem Warenkorb entfernt.',
+                'inactive-add' => 'Ein inactiver Artikel kann nicht zum Warenkorb hinzugefügt werde.',
```

Comment:
German and English mix-up in "inactiver" => "ina**k**tiver"

Reply:
I fixed this typo.